linaro/linaro-4.9.4.cmake

```diff
@@ -0,0 +1,6 @@
+set(CMAKE_SYSTEM_NAME Linux)
+
+set(CMAKE_C_COMPILER   "${CMAKE_CURRENT_SOURCE_DIR}/linaro-4.9.4/bin/arm-linux-gnueabihf-gcc")
+set(CMAKE_CXX_COMPILER "${CMAKE_CURRENT_SOURCE_DIR}/linaro-4.9.4/bin/arm-linux-gnueabihf-g++")
```

Comment:
I would change the references to `CMAKE_CURRENT_SOURCE_DIR` to an environment variable defined by the docker image. The CMake syntax for referencing an environment variable is `$ENV{FOOBAR}`, where `FOOBAR` is of course the name of the environment variable.